pkg/skaffold/build/local/local.go

```diff
@@ -123,8 +126,8 @@ func (b *Builder) buildArtifact(ctx context.Context, out io.Writer, a *latest.Ar
 }
 
 func (b *Builder) runBuildForArtifact(ctx context.Context, out io.Writer, a *latest.Artifact, tag string, platforms platform.Matcher) (string, error) {
-	if !b.pushImages {
-		// All of the builders will rely on a local Docker:
+	if !b.buildx && !b.pushImages {
+		// Most builders will rely on a local Docker (except when using a remote buildkit via buildx):
```

Comment:
How does this relate to using a local buildkit? Will these have to basically dump to local docker?

Reply:
If there is a local docker daemon, it should have a "default" buildkit builder.
If I understand correctly, it is currently being used when the skaffold.yaml has `useBuildkit: true`.
It can be used with buildx too, executing `skaffold config set buildx-builder default`.
Let me know if I understood ok the question or more info is needed.